### ISSUE TITLE
Appveyor Exporting Exe

### ISSUE THREAD

**Issue by maddie-wang** [jitsi/jitsi-meet-electron#360]
Hello! 

Thank you @saghul  for answering https://github.com/jitsi/jitsi-meet-electron/issues/353!

I appreciate it a lot and I got it to build successfully using the online interface Appveyor.com

I was wondering how you get the windows artifact (exe file) from Appveyor? I looked through the official documentation and I tried a bunch of things (like git commit on deploy, some artifact stuff), but nothing seems to work. I would love to hear how you get the Jitisi meet exe build from Appveyor!

I love this project and can't wait to hear back

**Comment by saghul** [comment]
As you can see our appveyor file is very simple: https://github.com/jitsi/jitsi-meet-electron/blob/master/appveyor.yml

The magic is done by electron-builder, so you don’t have to do anything :-)

Just push a tag to the repo and when appveyor runs it will detect it’s for a tag and automatically upload the artifacts.

**Comment by Echolon** [comment]
@maddie-wang I will close this as the question has been answered to my understanding.

**Comment by maddie-wang** [comment]
thank you @saghul  for responding so quickly!
I appreciate your help because after hours of frustration, i tried your solution and my face lit up as soon as I saw the exe!! 


For everyone else wanting step-by-step, here's what worked for me.

connect the git repo to appveyor.com. make sure you add "GH_TOKEN" env variable in the settings on appveyor.com. 

pick the next version number. for this example, I picked 3.0.2

change package.json version to 3.0.2
git commit ....
git tag -a v3.0.2 -m "v3.0.2"         
git push origin v3.0.2

check your repo on github under "releases" and you'll see the exe there!
